policyengine/monitoring/watched_reforms.md

```diff
@@ -1,7 +1,7 @@
 ## Autumn Budget UC reform
-* gini_change: -0.004395061661465764
-* loser_share: 0.0
-* net_cost: £2.82bn
-* net_cost_numeric: 2822803143.6693115
-* poverty_change: -0.018696715809341735
-* winner_share: 0.11899752178044937
+
+* Net cost: £2.81bn
+* Poverty change: -2.0%
```

Comment:
Looks like this wasn't updated before? It's [2.0% today](https://policyengine.org/uk/population-impact?UC_reduction_rate=55&UC_work_allowance_without_housing=557&UC_work_allowance_with_housing=335), not 1.9% as this replaces.

Reply:
Thanks, yeah I should figure out how to run this automatically in a GH action and commit it.